src/app/store/product-detail/product-detail.component.html

```diff
@@ -0,0 +1,58 @@
+<div fxLayout="row">
+  <mat-card fxFlex="60" fxFlex.md="100" fxFlex.sm="100" fxFlex.xs="100">
+    <mat-spinner *ngIf="loading"></mat-spinner>
+    <div fxLayout="row" fxLayoutAlign="center">
+    <img mat-card-image [src]="product.imageUrl" alt="{{ product.name }}" >
+  </div>
+    <mat-card-content>
+      <div fxLayout="row" fxLayoutAlign="space-between">
+        <div>
+          <h3>{{ product.name }}</h3>
+        </div>
+        <div>
+          <h3>{{ product.price }}</h3>
+        </div>
+      </div>
+      <div class="mat-small">
+        {{ product.description }}
+      </div>
+      <br>
+      <mat-divider></mat-divider>
+      <br>
+      <form [formGroup]="shoppingForm" (ngSubmit)="addToCart()" novalidate>
+        <div *ngIf="product.sizes?.length > 0" fxLayout="row" fxLayoutAlign="space-between">
+          <div fxLayout="column">
+            <h3>Select Size</h3>
+            <div fxLayout="row">
+              <mat-button-toggle-group formControlName="size">
+                <div *ngFor="let size of product.sizes">
+                  <mat-button-toggle type="button" value='{{ size }}'>{{ size }}</mat-button-toggle>
+                </div>
+              </mat-button-toggle-group>
+            </div>
+          </div>
+        </div>
+        <div  fxLayout="row" fxLayoutAlign="space-between">
+          <div fxLayout="column">
+          <h3>Quantity</h3>
```

Comment:
this should probably be a label tag with a for attribute identifying the id of the associated input